docs/data/date-pickers/date-field/DebouncedDateField.tsx

```diff
@@ -8,10 +8,10 @@ import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
 import Button from '@mui/material/Button';
 
-const today = dayjs();
+const today = dayjs('2022-04-07');
 
 export default function DebouncedDateField() {
-  const [value, setValue] = React.useState<Dayjs | null>(() => today.add(1, 'week'));
+  const [value, setValue] = React.useState<Dayjs | null>(dayjs('2022-04-07'));
```

Comment:
Was there any issues leaving the previous logic intact in here? 🤔 
This changes how the demo behaves.
```suggestion
  const [value, setValue] = React.useState<Dayjs | null>(today.add(1, 'week'));
```

Reply:
Why does it change the demo behavior? The goal was to set the date to something which is not today. Such that you can click on the "Set to today" button with a delay of 500ms

The problem is the same as others. If you put the default to `today.add(1, 'week')` nextjs will prepare the html for 15 of September. But tomorrow hydratation will override it with the 16 of September

Reply:
> Why does it change the demo behavior?

I meant that now once you open this demo, the `set to today` is enabled, because the selected day differs from `today`, but in your proposal it is disabled as `today` === `selected date`.
![Screenshot 2022-09-08 at 12 52 40](https://user-images.githubusercontent.com/4941090/189092906-4b8b78cc-9307-458a-87b9-673698d0e843.png)


> The problem is the same as others. If you put the default to today.add(1, 'week') nextjs will prepare the html for 15 of September. But tomorrow hydratation will override it with the 16 of September

`today` is set to `2022-04-07`, should it really ever hydrate to anything than `2022-04-14`? 🤔

Reply:
But if we use dynamic `today` , won't this demo remain with the hydration issue? 🤔 
Maybe using a different CTA would be better?
Like `Reset date` or something along these lines?

Reply:
Is `today` the problem of is it to display an HTML element based on `today` value?

But yes of course fixing all the date with another cal to action does the job

Reply:
The problem is a difference in the rendered UI at least based on errors being thrown locally as well, hence, if we will make sure the button `disabled` flag always renders as `false` - it should be all good. 👌